src/pages/auth/PrivateViews.js

```diff
@@ -1,3 +1,12 @@
-export default function PrivateViews({ children }) {
-  return children;
+import React, { Fragment } from 'react';
+import NavBar from 'pages/dashboard/components/NavBar';
+import { navConfiguration } from 'config/navigation';
+
+export default function PrivateViews({ children, ...rest }) {
+  return (
+    <Fragment>
+      <NavBar configuration={navConfiguration} {...rest} />
+      <main className="min-h-screen bg-secondary-lightest">{children}</main>
```

Comment:
We have to make this have in `md` resolutions, for example, `margin-left: 100px` (same size than width of the navbar for `md` resolutions)